packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -21,36 +21,84 @@ export const SidePanel = ({
   panelClassNameOverrides = '',
   scrollable = true
 }: SidePanelProps) => {
+  const [isClosing, setIsClosing] = useState(false)
```

Comment:
All the transition + transition logic taken from the tx history reskin. It's exactly the same and will reduce diff there.